website/docs/components/link/standalone/index.md

```diff
@@ -3,7 +3,7 @@ title: Standalone Link
 description: A link used in isolation and not as a part of surrounding body text.
 caption: A link used in isolation and not as a part of surrounding body text.
 links:
-  figma: https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?node-id=2365%3A21590&t=XC8SUxxJOFHgqYzK-1
+  figma: hhttps://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?node-id=67421-84416&t=w8xQlWxzH7bwXLe2-1
```

Comment:
[blocker] typo (there may be others, please double-check)
```suggestion
  figma: https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?node-id=67421-84416&t=w8xQlWxzH7bwXLe2-1
```

Reply:
Fixed!